### ISSUE TITLE
баг кэша???

### ISSUE THREAD

**Issue by STyulb** [vtyulb/BSA-Analytics#311]
Время от времени несколько или один луч в отдельных блоках имеют неожиданно большую шумовую дорожку. Так как будто сложилось всего несколько записей из более чем 1000. Пример (нижняя площадка): блок 170 мод.2 луч7; блок 337 (почти весь)

**Comment by vtyulb** [comment]
Да, это что-то с кешем. Поставлю на ночь пересчитываться, если не забуду
